Assets/Samples/GamepadMouseCursor/VirtualMouseInput.cs

```diff
@@ -153,11 +159,38 @@ public float scrollSpeed
         /// </remarks>
         public Mouse virtualMouse => m_VirtualMouse;
 
+        /// <summary>
+        /// If <see cref="cursorMode"/> is set to <see cref="CursorMode.HardwareCursorIfAvailable"/>, whether to
+        /// disable the system mouse, if present. This is on by default.
+        /// </summary>
+        /// <value>If true, the system mouse will be disabled if present and while the VirtualMouseInput component is active.</value>
+        /// <remarks>
+        /// If the component drives the system mouse cursor rather than using a software cursor, the system mouse cursor position will
+        /// get warped around using <see cref="Mouse.WarpCursorPosition"/>. This, however, has the side-effect of generating input
+        /// on the system <see cref="Mouse"/> device. In other words, mouse motion from the gamepad will come in through both
+        /// the VirtualMouse created for the gamepad and the <see cref="Mouse"/> added by the system (with a one frame lag for
+        /// the latter).
+        ///
+        /// To avoid this, the system mouse will by default get disabled while the VirtualMouseInput component is enabled. Thus,
+        /// the system <see cref="Mouse"/> will not receive input while the VirtualMouseInput component is enabled. The idea here
+        /// is that the application should determine when the gamepad is used and only turn on the gamepad cursor while this is
+        /// the case. One possible approach is to set up a control scheme for gamepads and enable the gamepad mouse cursor only
+        /// while the gamepad scheme is active.
+        ///
+        /// However, by setting this property to false, the disabling of the system <see cref="Mouse"/> device can be suppressed.
+        /// This means the system mouse will stay fully functional. You will, however, see concurrent input on both the VirtualMouse
+        /// and the system mouse. This may, for example, interfere with automatic control scheme switching (which is why the disabling
+        /// is on by default).
+        /// </remarks>
+        public bool disableSystemMouse
```

Comment:
A user got caught off guard that the system mouse wouldn't work if VirtualMouseInput is used. This is intentional and IMO the better thing to do in the current setup. However, I can see how that can be quite undesirable so I opted for just exposing an option to toggle the behavior off. Default is on so behavior is unchanged from before.